compiler/src/iree/compiler/Codegen/Utils/MarkerUtils.h

```diff
@@ -37,6 +37,9 @@ StringRef getWorkgroupL1TileMarker();
 /// Workgroup memory.
 StringRef getCopyToWorkgroupMemoryMarker();
 
+/// Marker for transpose padding operations.
+StringRef getTransposePadMarker();
```

Comment:
if it is only used internally to the pass you don't need to add the marker here, it is better to keep it as a static in the pass